yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/AddressTransformer.java

```diff
@@ -0,0 +1,42 @@
+package com.yoti.api.client.spi.remote;
+
+import java.util.Map;
+
+import com.yoti.api.attributes.AttributeConstants.HumanProfileAttributes;
+import com.yoti.api.client.Attribute;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class AddressTransformer {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AttributeListConverter.class);
+
+    private AddressTransformer() {}
+
+    static final AddressTransformer newInstance() {
+        return new AddressTransformer();
+    }
+
+    public Attribute<String> transform(Attribute<?> structuredAddress) {
+        Attribute<String> transformedAddress = null;
+        try {
+            Attribute<Map<?, ?>> address = (Attribute<Map<?, ?>>) structuredAddress;
```

Comment:
I've amended it.  We need to have a cast somewhere, so that we get the _formatted_address_ value out of the map.
Java compiler should inline the local variables where it can anyway.